src/config.py

```diff
@@ -1,7 +1,11 @@
 import json
+import urllib.parse, hashlib
 
 config_json = {}
 
+default_avatar_folder_path = "/static/images/"
```

Comment:
Per [PEP 8](https://www.python.org/dev/peps/pep-0008/#constants) style, use CAPS for constant naming.